nepal_compliance/nepal_compliance/report/sales_return_vat_register/sales_return_vat_register.py

```diff
@@ -174,12 +158,17 @@ def execute(filters=None):
 		taxes = frappe.db.get_all("Sales Taxes and Charges", filters={"parent":sale.name}, fields=['*'])
 		total_qty = 0
 		total_rate = 0
-		net_total = sale.net_total if sale.net_total else 0
+		vat = 0
+		tds = 0
+		invoice_total = sale.grand_total
+		net_total = sale.net_total if sale.net_total else invoice_total
 		for item in items:
 			total_qty += item.qty	
 			total_rate += item.rate
 		for tax in taxes:
-			vat = tax.rate if tax.rate == 13 else ''
-			tds = tax.rate if (tax.rate == 1.5 or tax.rate == 15) else ''
-			data.append([sale.posting_date, sale.nepali_date, sale.name, sale.customer, sale.customer_group, sale.customer_address, sale.project, sale.cost_center, '', sale.owner, total_qty, total_rate, '',sale.discount_amount, net_total, tax.charge_type, tax.account_head, vat, tds, sale.total_taxes_and_charges, sale.grand_total, sale.rounded_total, sale.total_advance, sale.outstanding_amount, sale.return_against])
-	return columns, data
\ No newline at end of file
+			if tax.rate == 13:
+				vat = tax.tax_amount
+			elif tax.rate in [1.5, 15]:
+				tds = tax.tax_amount
+		data.append([sale.posting_date, sale.nepali_date, sale.name, sale.customer, sale.customer_group, sale.project, sale.cost_center, sale.vat_number, sale.owner, total_qty, sale.total,sale.discount_amount, sale.total, net_total, vat, tds, sale.total_taxes_and_charges, sale.total_advance, sale.grand_total, sale.rounded_total, sale.outstanding_amount, sale.return_against])
+	return columns, data
```

Comment:
The logic for calculating `vat`, `tds` and `tax.rate` in the `sales_return_vat_register.py` file has been modified. Please update the PR description to reflect the summary of the changes.